packages/svelte-ux/src/lib/components/SelectField.svelte

```diff
@@ -352,10 +374,15 @@
     on:blur={onBlur}
     on:keydown={onKeyDown}
     on:keypress={onKeyPress}
-    actions={(node) => [selectOnFocus(node)]}
-    class={cls('h-full', theme.field, classes.field)}
+    actions={fieldActions}
+    classes={{
+      root: cls('h-full'),
```

Comment:
`cls()` isn't needed since we are not doing any overrides with later parameters (or non-string values like objects and arrays)

Reply:
Reverted this tidbit.